.github/workflows/build.yml

```
@@ -19,9 +19,9 @@ jobs:
           - os: windows-latest
             tasks: mingwX64Test
           - os: macos-13
-            tasks: macosX64Test iosX64Test
+            tasks: macosX64Test iosX64Test tvosX64Test watchosX64Test
           - os: macos-latest
-            tasks: macosArm64Test iosSimulatorArm64Test
+            tasks: macosArm64Test iosSimulatorArm64Test tvosX64Test watchosSimulatorArm64Test
```

Should this be this?

```suggestion
            tasks: macosArm64Test iosSimulatorArm64Test tvosSimulatorArm64Test watchosSimulatorArm64Test
```